src/gui/userinfo.cpp

```diff
@@ -123,7 +123,7 @@ void UserInfo::slotUpdateLastInfo(const QJsonDocument &json)
 
     // Quota
     auto objQuota = objData.value("quota").toObject();
-    auto quota = objQuota.value("quota").toInt();
+    qint64 quota = objQuota.value("quota").toDouble();
```

Comment:
Should be `toLongLong()` really and then it can be auto.

Reply:
I was about to change it but there is no toLong()

Reply:
Ah yes... I had this in mind: https://doc.qt.io/qt-5/qvariant.html#toLongLong

But it's a pesky `QJsonValue` and they didn't add a `toLongLong()` to that one which... makes no sense to me, either you have `toInt()` and `toLongLong()` or none of them.

Let's keep it like this (other approach would be `.toVariant().toLongLong()` but that wouldn't buy us much...